plugins/versionpress/setup-hooks.php

```diff
@@ -719,7 +719,7 @@ function vp_admin_menu()
         'manage_options',
         'versionpress',
         'versionpress_page',
-        null,
+        'dashicons-backup,
```

Comment:
Missing apostrophe at the end.

Reply:
Oh! my bad, this happens when jumping to file directly on GitHub 🤦🏻‍♂️ 
fixed in 1fb585263317c5c234749d9fb9dfd0b089a8dcfb